GreenhouseNode/GreenhouseNode/links/PrintShared.c

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
I don't see the point in this class, you can just use `printf("")` inside the other implementations.